filelink.rb

```diff
@@ -0,0 +1,15 @@
+module FileLink
+
+  def file_link.write(name, desg, dept)
+    file_name = "employeedetails.csv"
```

Comment:
dont hard code the file name. Its not reusable anymore. 
If you want to write to a file for another class. 
```
module FileUtilities
def write_to_csv(object, filename)
  CSV.open(filename, "a") do |file| 
    file << obj.keys
  end
end
```